src/test/resources/features/createEvent.feature

```diff
@@ -22,6 +22,20 @@ Feature: Events
       And I wait 5 seconds to avoid Too Many Requests message
 
   Scenario: I can open my created events
-    When I select event "EventTitle"
-    Then I validate the Created By name is the same as the profile account
-      And I validate the page title is "EventTitle"
\ No newline at end of file
+    Given I select event "EventTitle"
+    When I validate the Created By name is the same as the profile account
+    Then I validate the page title is "EventTitle"
+    And I wait 5 seconds to avoid Too Many Requests message
+
+  Scenario: I delete created events
+    Given I select event "EventTitle"
```

Comment:
instead of Given should not it be When? do you have a background in this feature? if yes, the "given" is already used in background, so we cannot have more than one given, when then